lib/ansible/plugins/connection/winrm.py

```diff
@@ -193,6 +195,13 @@ def __init__(self, *args, **kwargs):
 
         super(Connection, self).__init__(*args, **kwargs)
 
+        if C.DEFAULT_DEBUG:
```

Comment:
Same as the comment in psrp, shouldn't we just set to INFO when not C.DEFAULT_DEBUG

Reply:
Fixed.